Gemfile

```diff
@@ -26,6 +26,8 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 4.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+# 
+gem 'rubocop', '~> 0.59.2'
```

Comment:
You, should not have this change here. You have this twice

Reply:
I agree